membrane_curvature/base.py

```diff
@@ -0,0 +1,126 @@
+"""
+MembraneCurvature
+=======================================
+
+:Author: Estefania Barreto-Ojeda
+:Year: 2021
+:Copyright: GNU Public License v3
+
+MembraneCurvature calculates the mean and Gaussian curvature of
+surfaces derived from a selection of reference.
+"""
+
+import numpy as np
+import warnings
+from membrane_curvature.surface import get_z_surface
+from membrane_curvature.curvature import mean_curvature, gaussian_curvature
+
+
+from MDAnalysis.analysis.base import AnalysisBase
+import MDAnalysis as mda
+
+
+class MembraneCurvature(AnalysisBase):
+
+    r"""
+    MembraneCurvature is a tool to calculate membrane curvature.
+
+    Parameters
+    ----------
+    universe : Universe or AtomGroup
+        An MDAnalysis Universe object.
+    select : str or iterable of str, optional. Default: 'all'
+        The selection string of an atom selection to use as a
+        reference to derive a surface.
+    pbc : bool, optional. Default: 'True'
+        Apply periodic boundary conditions.
+    n_x_bins : int, optional. Deafult: '100'
+        Number of bins in grid in the `x` dimension.
```

Comment:
We use single backquotes when referring to a parameter. For anything else we either use explicit reST roles (like :func: or :class: or :attr:) or use "inline code" _double_ backquotes
```suggestion
        Number of bins in grid in the ``x`` dimension.
```
ALternatively, you can also use the :math: role
```reST
        Number of bins in grid in the :math:`x` dimension.
```
although in this specific case, this seems less readable and you could also just write
```reST
        Number of bins in grid in the x dimension.
```

The main point is that when you mark-up something then think about why you are marking it up and what _meaning_ it should carry.